### ISSUE TITLE
Import lombscargle from astropy.stats will be deprecated.

### ISSUE THREAD

**Issue by ojhall94** [KeplerGO/lightkurve#535]
Currently receiving the following warning when using lightkurve.periodogram:

>WARNING: AstropyDeprecationWarning: Importing LombScargle from astropy.stats has been deprecated and will no longer be supported in future.
Please import this class from the astropy.timeseries module instead [astropy.stats.lombscargle]

We should probably fix this before it happens :)

**Comment by barentsen** [comment]
Thank you for reporting this @ojhall94!!  I think the best backward-compatible solution involves something like this:
```python
try:
    from astropy.timeseries import LombScargle
except ImportError:
    from astropy.stats import LombScargle
```

I opened #536 to implement this fix!

**Comment by barentsen** [comment]
I'm closing this issue because I merged #536, however please do re-open if the deprecation warning appears again when using the latest master branch!
